<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppuntamentiModulo</class>
 <widget class="QDialog" name="AppuntamentiModulo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#AppuntamentiModulo{
background-color:qlineargradient(spread:pad, x1:0.058, y1:0.136364, x2:1, y2:1, stop:0 rgba(0, 255, 127, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QLineEdit" name="nomeLe">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cognomeLe">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Cognome</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Anno</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Mese</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Giorno</string>
   </property>
  </widget>
  <widget class="QComboBox" name="meseCombo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>01</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>02</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>03</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>04</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>05</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>06</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>07</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>08</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>09</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="giornoCombo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="annoCombo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cfLe">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Codice Fiscale</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inviaformBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenota</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tamponeCombo">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Tipo di tampone</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/back.png</normaloff>img/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
